ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolSpecBuilder.java

```diff
@@ -320,6 +321,7 @@
         evm,
         transactionValidator,
         transactionProcessor,
+        privateTransactionProcessor,
```

Comment:
Any side effects if this is null (when privacy is disabled)?

Reply:
There shouldn't be any ...